_posts/2019-08-01-gcc.md

```diff
@@ -0,0 +1,114 @@
+---
+site: freiburg
+title: 'Galaxy Community Conference - a brief summary!'
+tags: [training, tiaas, gcc]
+supporters:
+- galaxy-europe
+- tiaas
+- denbi
+- elixir
+---
+
+
+The 2019 Galaxy Community Conference (GCC2019) was held in Freiburg, Germany, 1-8 July.
+There was one day of training, three days of mixed meeting and training, and then two to four days of CollaborationFest.
+The meeting featured joint & parallel sessions; invited, accepted, and lightning talks; poster and demo sessions;
+birds-of-a-feather meetups; social events; and training sessions every day.
+
+
+![GCC2019 Logo](/assets/media/gcc2019/gcc_logo_blue_bg.png)
+![Group Picture](/assets/media/gcc2019/grouppicture.jpg)
+
+With 231 participants, GCC2019 was the largest GCC ever with 39 Galaxy training courses, 65 talks, 54 posters,
+13 demos and more than 30 different BoFs. Our participants came from 27 different countries of Eastern- and Western Europe,
+North America, Asia, Oceania and Sub-saharan Africa.
+
+![Group Picture](/assets/media/gcc2019/GCC2019_stats.png)
+
+GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation and Computational Chemistry.
```

Comment:
```suggestion
GCC2019 has started on Monday, 1st July, with a dedicated training day for Galaxy beginners offering a user-centric track with “Introduction to using Galaxy” in parallel to the admin and developer focused “Introduction to Galaxy Administration”. Participants interested in long read sequencing and some wet-lab work could visit and try the Nanopore loading zone where the Oxford Nanopore MinION was presented. In the evening, the first Birds-of-a-Feather (BoFs) were offered for special interest groups on Single Cell data analysis, Genome Annotation, and Computational Chemistry.
```

Reply:
Sorry, can not take this over, merged it to late :(